src/addons/presenter.js

```diff
@@ -89,7 +92,8 @@ inherit(Presenter, PureComponent, {
     return (
       createElement(PresenterMediator, {
         presenter   : this,
-        parentState : this.state
+        parentState : this.state,
+        parentProps : this.props
```

Comment:
That's where I figured the change would go, but I was having a hard time figuring out how `parentProps` and `parentState` actually get used and passed down. They get passed to a mediator component as props... then what?